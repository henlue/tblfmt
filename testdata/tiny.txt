===============================================
format: aligned
===============================================
 z 
---
 x 
(1 row)

===============================================
format: aligned
linestyle: ascii
border: 0
===============================================
z 
-
x 
(1 row)

===============================================
format: aligned
linestyle: ascii
border: 1
===============================================
 z 
---
 x 
(1 row)

===============================================
format: aligned
linestyle: ascii
border: 2
===============================================
+---+
| z |
+---+
| x |
+---+
(1 row)

===============================================
format: aligned
linestyle: old-ascii
border: 0
===============================================
z 
-
x 
(1 row)

===============================================
format: aligned
linestyle: old-ascii
border: 1
===============================================
 z 
---
 x 
(1 row)

===============================================
format: aligned
linestyle: old-ascii
border: 2
===============================================
+---+
| z |
+---+
| x |
+---+
(1 row)

===============================================
format: aligned
linestyle: unicode
border: 0
===============================================
z 
─
x 
(1 row)

===============================================
format: aligned
linestyle: unicode
border: 1
===============================================
 z 
───
 x 
(1 row)

===============================================
format: aligned
linestyle: unicode
border: 2
===============================================
┌───┐
│ z │
├───┤
│ x │
└───┘
(1 row)

===============================================
format: aligned
linestyle: unicode-double
border: 0
===============================================
z 
═
x 
(1 row)

===============================================
format: aligned
linestyle: unicode-double
border: 1
===============================================
 z 
═══
 x 
(1 row)

===============================================
format: aligned
linestyle: unicode-double
border: 2
===============================================
╔═══╗
║ z ║
╠═══╣
║ x ║
╚═══╝
(1 row)

===============================================
format: expanded
===============================================
-[ RECORD 1 ]-
 z | x 

===============================================
format: expanded
linestyle: ascii
border: 0
===============================================
* Record 1 
z x 

===============================================
format: expanded
linestyle: ascii
border: 1
===============================================
-[ RECORD 1 ]-
 z | x 

===============================================
format: expanded
linestyle: ascii
border: 2
===============================================
+-[ RECORD 1 ]-+
| z | x        |
+---+----------+

===============================================
format: expanded
linestyle: old-ascii
border: 0
===============================================
* Record 1 
z x 

===============================================
format: expanded
linestyle: old-ascii
border: 1
===============================================
-[ RECORD 1 ]-
 z | x 

===============================================
format: expanded
linestyle: old-ascii
border: 2
===============================================
+-[ RECORD 1 ]-+
| z | x        |
+---+----------+

===============================================
format: expanded
linestyle: unicode
border: 0
===============================================
* Record 1 
z x 

===============================================
format: expanded
linestyle: unicode
border: 1
===============================================
─[ RECORD 1 ]─
 z │ x 

===============================================
format: expanded
linestyle: unicode
border: 2
===============================================
┌─[ RECORD 1 ]─┐
│ z │ x        │
└───┴──────────┘

===============================================
format: expanded
linestyle: unicode-double
border: 0
===============================================
* Record 1 
z x 

===============================================
format: expanded
linestyle: unicode-double
border: 1
===============================================
═[ RECORD 1 ]═
 z ║ x 

===============================================
format: expanded
linestyle: unicode-double
border: 2
===============================================
╔═[ RECORD 1 ]═╗
║ z ║ x        ║
╚═══╩══════════╝

===============================================
format: json
===============================================
[{"z":"x"}]
===============================================
format: unaligned
===============================================
z
x

===============================================
format: csv
===============================================
z
x

===============================================
format: html
===============================================
<table>
  <caption></caption>
  <thead>
    <tr>
      <th align="left">z</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td align="left">x</td>
    </tr>
  </tbody>
</table>

===============================================
format: asciidoc
===============================================
[%header]
|===
|z
|x
|===

