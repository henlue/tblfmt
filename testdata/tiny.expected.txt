+-[ RECORD 1 ]-+
| z | x        |
+---+----------+


